components/prize_pool/commons/prize_pool.lua

```diff
@@ -0,0 +1,450 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:PrizePool
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent') -- Note: This can be overwritten
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local WidgetInjector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+
+local PrizePool = Class.new(function(self, ...) self:init(...) end)
+local Placement = Class.new(function(self, ...) self:init(...) end)
+
+local TODAY = os.date('%Y-%m-%d')
```

Comment:
```suggestion
local _TODAY = os.date('%Y-%m-%d')
```
as per our notation, right?

Reply:
only on groundhog day

Reply:
https://github.com/Liquipedia/Lua-Modules/wiki/Style-guide-for-core-Lua-modules#naming

Reply:
i meant - is today a constant?

Reply:
You're right in that calling it const is wrong. ReadOnly Module global? :D